zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -410,7 +410,7 @@ protected boolean processZKCmd(MyCommandOptions co) throws CliException, IOExcep
             history.put(commandCount, history.get(i));
             processCmd(cl);
         } else if (cmd.equals("history")) {
-            for (int i = commandCount - 10; i <= commandCount; ++i) {
+            for (int i = commandCount - 9; i <= commandCount; ++i) {
                 if (i < 0) {
```

Comment:
Look at an example from linux cmd `history`, don't need to include the recent one: `history`
`for (int i = commandCount - 10; i < commandCount; ++i)` ?

Reply:
Thanks for the review @maoling. I have made the changes so as not to include history as a recent cmd.